### ISSUE TITLE
fix website connection (https is broken)

### ISSUE THREAD

**Issue by alltheseas** [stemstr/Client#129]
<img width="788" alt="Screenshot 2023-06-28 at 1 00 56 AM" src="https://github.com/stemstr/Client/assets/64376233/570379fe-8c95-4305-988c-b57cb1fc944f">

**Comment by SamSamskies** [comment]
yep, it's only working when prefixed with www https://www.stemstr.app/
